clients/fides-js/.eslintrc.json

```diff
@@ -2,12 +2,15 @@
   "extends": [
     "airbnb",
     "airbnb-typescript/base",
-    "prettier"
+    "prettier",
+    "standard",
+    "standard-preact"
```

Comment:
FYI: I started tinkering with this branch and the eslint configs you added (`preact`, `standard`, `standard-preact`) caused my linter to crash and not obey the Typescript tsconfig settings... so I removed them for now 🙃 

I think you were probably just experimenting with these anyways